src/main/kotlin/org/rust/ide/formatter/impl/spacing.kt

```diff
@@ -254,8 +259,10 @@ private fun SpacingContext.blockMustBeMultiLine(): Boolean {
     }
 }
 
+/**
+ * Assumes that left is before right and they are siblings, otherwise bad things will happen.
+ */
 private fun countNonWhitespaceASTNodesBetween(left: ASTNode, right: ASTNode): Int {
-    // TODO(jajakobyly): assert that left is before right and they are siblings
```

Comment:
TIL about Kotlin assert :) 

https://youtrack.jetbrains.com/issue/KT-12454
https://youtrack.jetbrains.com/issue/KT-7540

One more reason to prefer always on assertions :)